### PR TITLE
Fixes #26901 - Use content certs for deb repos

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

**Comment by jlsherrill** [comment]
[test katello]

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/views/new-repository.html [hunk 1]
```diff
@@ -251,7 +251,7 @@ <h4 ng-show="repository.content_type !== undefined" translate> Published Reposit
         </select>
       </div>
 
-      <div bst-form-group label="{{ 'SSL Client Key' | translate }}" ng-show="repository.content_type === 'yum'">
+      <div bst-form-group label="{{ 'SSL Client Key' | translate }}" ng-show="repository.content_type === 'yum' || repository.content_type === 'deb'">
```
Comment (jlsherrill) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/views/new-repository.html]: Looking at the backend code, it seems that this is supported on all types, what to just open it up to all?

Comment (mdellweg) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/views/new-repository.html]: If you think so...

**Comment by mdellweg** [comment]
@jlsherrill ping

**Pull request merged**